src/Files.Backend/Helpers/NativeFindStorageItemHelper.cs

```diff
@@ -82,25 +84,40 @@ public struct WIN32_FIND_DATA
 		public const int FIND_FIRST_EX_LARGE_FETCH = 2;
 
 		[DllImport("api-ms-win-core-file-l1-1-0.dll", CharSet = CharSet.Unicode)]
-		public static extern bool FindNextFile(IntPtr hFindFile, out WIN32_FIND_DATA lpFindFileData);
+		public static extern bool FindNextFile(
+			IntPtr hFindFile,
+			out WIN32_FIND_DATA lpFindFileData);
 
 		[DllImport("api-ms-win-core-file-l1-1-0.dll")]
-		public static extern bool FindClose(IntPtr hFindFile);
+		public static extern bool FindClose(
+			IntPtr hFindFile);
 
 		[DllImport("api-ms-win-core-timezone-l1-1-0.dll", SetLastError = true)]
-		public static extern bool FileTimeToSystemTime(ref FILETIME lpFileTime, out SYSTEMTIME lpSystemTime);
+		public static extern bool FileTimeToSystemTime(
+			ref FILETIME lpFileTime,
+			out SYSTEMTIME lpSystemTime);
 
-		public static bool GetWin32FindDataForPath(string targetPath, out WIN32_FIND_DATA findData)
+		public static bool GetWin32FindDataForPath(
+			string targetPath,
+			out WIN32_FIND_DATA findData)
 		{
 			FINDEX_INFO_LEVELS findInfoLevel = FINDEX_INFO_LEVELS.FindExInfoBasic;
+
 			int additionalFlags = FIND_FIRST_EX_LARGE_FETCH;
 
-			IntPtr hFile = FindFirstFileExFromApp(targetPath, findInfoLevel, out findData, FINDEX_SEARCH_OPS.FindExSearchNameMatch, IntPtr.Zero, additionalFlags);
+			IntPtr hFile = FindFirstFileExFromApp(
+				targetPath,
+				findInfoLevel,
+				out findData, FINDEX_SEARCH_OPS.FindExSearchNameMatch,
+				IntPtr.Zero, additionalFlags);
```

Comment:
I think you should split these two lines into four

Reply:
Fixed